webapp/components/Notification/Notification.vue

```diff
@@ -69,10 +69,9 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-30T07:40:58Z" title="Wednesday, October 30th 2019, 8:40:58 am +01:00">Oct 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -69,7 +69,7 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss" scoped>
```


Do we need it scoped here? Are we defining the `.notification` class anywhere else?

Whenever possible I would go for the not scoped option because `scoped` adds an attribute to the HTML and an attribute selector to each of our CSS selectors – so `.notification.read` becomes `.notification.read[data-123456]` which is more expensive than a simple class selector. Here's [a good explanation](https://vue-loader.vuejs.org/guide/scoped-css.html#scoped-css).